ckan/lib/helpers.py

```diff
@@ -2344,6 +2344,35 @@ def mail_to(email_address, name):
     return html
 
 
+@core_helper
+def get_base_public_folder(app_conf):
```

Comment:
this is something site-configuration related and some of the extensions need it but it seems silly to add to the plugins toolkit. How about just loading the default into the config object when reading the config?